profile/profile.go

```diff
@@ -394,7 +394,7 @@ func (profile *Profile) UpdateMetadata(processName, binaryPath string) (changed
 	}
 
 	// Update LinkedPath if if differs from the process path.
-	// This will (at the moment) only be the case for the Portmaster profile.
+	// This will be the case for profiles that are assigned in a special way.
 	if profile.LinkedPath != binaryPath {
 		profile.LinkedPath = binaryPath
```

Comment:
Hm, if I ever hit this part while hunting a bug it would be nice to get reminded of those "special ways"